webdriver-spec.html

```diff
@@ -4098,6 +4103,23 @@
 </section> <!-- /Partial Link Text -->
 
 <section>
+<h4>Tag Name</h4>
+
+<p>To find a <a>web element</a> with the <dfn>Tag
+ Name</dfn> <a>strategy</a> the following steps need to be completed:
+
+<ol>
+ <li><p><p>Let <var>elements</var> be the result of
+ calling <a>getElementsByTagName</a> with
```

Comment:
Missing reference to https://dom.spec.whatwg.org/#dom-element-getelementsbytagname.